style(notes): improve layout and spacing in Notes component

tomer-shvadron opened the pull request:
- Adjust flex properties for size and alignment
- Add padding and overflow handling to improve user interface
- Update subproject commit reference in data-migrations


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Style**
  - Improved layout and scrolling behavior in the notes section for a more consistent and user-friendly experience.

- **Chores**
  - Updated internal data migration references to the latest version.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

tomer-shvadron committed:
style(notes): improve layout and spacing in Notes component

- Adjust flex properties for size and alignment
- Add padding and overflow handling to improve user interface
- Update subproject commit reference in data-migrations

Omri-Levy commented on apps/backoffice-v2/src/domains/notes/Notes.tsx:
```diff
@@ -85,7 +86,7 @@ export const Notes = ({
 
         <Separator />
 
-        <div className={`space-y-4 p-4`}>
```
If the intent is a scroll, use the scroll area component instead.

Omri-Levy commented on apps/backoffice-v2/src/domains/notes/Notes.tsx:
```diff
@@ -29,11 +29,12 @@ export const Notes = ({
       </div>
-      <div className={`flex flex-col gap-1 border-none`}>
+
+      <div className={`p-b-4 flex h-[calc(100%-48px)] flex-col gap-1 border-none`}>
```
+1

tomer-shvadron committed:
refactor(notes): improve layout and structure of notes component

- Replace div with ScrollArea for better scrolling experience
- Adjust padding and class names for improved styling
- Clean up imports by removing unused elements
